tests/1/infos_test/sql_like_query.md

```diff
@@ -1,11 +1,11 @@
 SELECT patient.gender, patient.name.given, patient.birthDate, condition.verificationStatus, practitioner.name
 
-FROM Condition as condition
-INNER JOIN Patient as patient
-ON condition.subject.reference = patient.id
-INNER JOIN Practitioner as practitioner
-ON condition.asserter.reference = practitioner.id
+FROM Condition AS condition
```

Comment:
can we have lower case? (maybe you can put everything and inputs in lower case in the parser to be case insensitive)

Reply:
I made some changes to support the fact that reserved SQL word are case-insensitive. Is-it what you were thinking?